.github/workflows/staging.yml

```diff
@@ -31,6 +31,12 @@ jobs:
         with:
           toolchain: nightly
 
+      - name: Cache target
+        uses: actions/cache@v2
+        with:
+          path: target
```

Comment:
I recommend using a fresh build for release. Any cache might cause a problem.

Is this only for staging?

Reply:
Okay, I agree

Reply:
What do you think about cache in release CI? If remove it from staging I think also remove it for release CI.

Reply:
You can keep it for staging.

But don't cache it for release.